colossalai/auto_parallel/tensor_shard/node_handler/strategy/batch_norm_generator.py

```diff
@@ -274,15 +306,15 @@ def generate(self):
         strategy_list.append(self.split_input_channel_1d(0, 1))
 
         # SR = SR x R WITH SYNC_BN
-        strategy_list.append(self.split_input_batch(0))
-        strategy_list.append(self.split_input_batch(1))
+        # strategy_list.append(self.split_input_batch(0))
+        # strategy_list.append(self.split_input_batch(1))
 
         # SS = SS x S WITH SYNC_BN
-        strategy_list.append(self.split_input_both_dim(0, 1))
-        strategy_list.append(self.split_input_both_dim(1, 0))
+        # strategy_list.append(self.split_input_both_dim(0, 1))
+        # strategy_list.append(self.split_input_both_dim(1, 0))
 
         # S01R = S01R x R WITH SYNC_BN
-        strategy_list.append(self.split_input_batch_1d(0, 1))
+        # strategy_list.append(self.split_input_batch_1d(0, 1))
```

Comment:
Why comment out these strategies?

Reply:
We need some additional pass to use SyncBN substitute BatchNorm, it will be done in future PR.

Reply:
OK, perhaps some TO-DOs and comments for this part of code will help us understand better.

Reply:
Sure, I will add TODOs  and comments to explain the reason those strategies are commented.